webpack/configs/prod.ts

```diff
@@ -0,0 +1,34 @@
+import * as UglifyJsPlugin from "uglifyjs-webpack-plugin";
+import * as webpack from "webpack";
+import * as merge from "webpack-merge";
+
+import CommonConfig from "./base";
+
+const ProdConfig: webpack.Configuration = {
+
+    mode: "production",
+
+    output: {
+        filename: "clarity.min.js"
+    },
+
+    plugins: [
+        new webpack.DefinePlugin({
```

Comment:
What does this mean and do?

Reply:
This sets environment variable to "production". We actually don't need this in clarity-js, since it's a node feature - removing

---
In reply to: [250239148](https://github.com/Microsoft/clarity-js/pull/126#discussion_r250239148) [](ancestors = 250239148)